plugins/tiddlywiki/filesystem/filesystemadaptor.js

```diff
@@ -124,7 +124,7 @@ FileSystemAdaptor.prototype.saveTiddler = function(tiddler,callback,options) {
 					return callback(err);
 				}
 			}
-			self.logger.log("Saved \'"+fileInfo.filepath+"\'");
+			self.logger.log("Saved \'"+fileInfo.filepath+"\'");debugger;
```

Comment:
I think, that's a problem.

Reply:
Still testing, will remove that, thanks.